src/utils/DockerUtil.js

```diff
@@ -20,20 +21,25 @@ export default {
       throw new Error('Falsy ip or name passed to docker client setup');
     }
 
-    let certDir = path.join(util.home(), '.docker/machine/machines/', name);
-    if (!fs.existsSync(certDir)) {
-      throw new Error('Certificate directory does not exist');
-    }
+    if (util.isLinux()) {
+      this.host = 'localhost';
+      this.client = new dockerode({socketPath: '/var/run/docker.sock'});
```

Comment:
How does this work? Isn't the docker daemon running as root? Do I need to run kitematic as root?